tests/src/texture/hipTextureRef2D.cpp

```diff
@@ -14,7 +14,9 @@ texture<float, 2, hipReadModeElementType> tex;
 bool testResult = true;
 
 __global__ void tex2DKernel(float* outputData,
+#ifdef __HIP_PLATFORM_HCC__

```

Comment:
@gargrahul - do we still need this different compilation paths on NVCC vs HCC ?  I thought with recent changes the only required change was "extern" before the texture global?